client/components/follow-button/test/button.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { render } from 'enzyme';
```

Comment:
ah, yeah, export the localized version as the default export and export the unlocalized version as a named export. then test the named export, supplying a fake translate function.